applications/CompressiblePotentialFlowApplication/custom_processes/define_2d_wake_process.h

```diff
@@ -0,0 +1,160 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Inigo Lopez
+//
+
+#if !defined(KRATOS_DEFINE_2D_WAKE_PROCESS_H_INCLUDED )
+#define  KRATOS_DEFINE_2D_WAKE_PROCESS_H_INCLUDED
+
+// Project includes
+#include "includes/model_part.h"
+#include "processes/process.h"
+#include "compressible_potential_flow_application_variables.h"
+
+namespace Kratos
+{
+
+///@name Kratos Classes
+///@{
+
+  /// Auxiliary process to define the wake in 2 dimensional problems.
+class KRATOS_API(COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) Define2DWakeProcess
+    : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of Define2DWakeProcess
+    KRATOS_CLASS_POINTER_DEFINITION(Define2DWakeProcess);
+
+    typedef Node<3> NodeType;
+    typedef ModelPart::NodeIterator NodeIteratorType;
+    typedef ModelPart::ElementIterator ElementIteratorType;
+    typedef typename std::size_t SizeType;
```

Comment:
I'm wondering why do you need the `typename` in here. Besides you are using this type declaration just in one place.